www/FabricPlugin.js

```diff
@@ -1,29 +1,16 @@
-'use strict';
+const exec = require('cordova/exec');
```

Comment:
This file should be JavaScript, but it now contains es6 code. It looks like the something went wrong while transpiling the code. This breaks the entire plugin on iOS.